app/controllers/Support.scala

```diff
@@ -20,10 +23,23 @@ import services.{AWS, Config, ImageMetadataService}
 
 import scala.concurrent.Future
 import scala.util.control.NonFatal
+import collection.JavaConverters._
 
 
 object Support extends Controller with PanDomainAuthActions {
 
+  private val capiPreviewCredentials: AWSCredentialsProvider = {
```

Comment:
Would it be possible to take this out of the repository rather than have a copy-paste?